ur_robot_driver/include/ur_robot_driver/ur/dashboard_client.h

```diff
@@ -80,7 +83,12 @@ class DashboardClient : public comm::TCPSocket
 protected:
   virtual bool open(int socket_fd, struct sockaddr* address, size_t address_len)
   {
-    return ::connect(socket_fd, address, address_len) == 0;
+    if (::connect(socket_fd, address, address_len) == -1)
+    {
+      LOG_ERROR("connect() error: %s", strerror(errno));
```

Comment:
Should these messages mention the calling class at least? So we can differentiate errors

Reply:
Sure